### ISSUE TITLE
[bug] focus behaviour 'ClickTo' and floating windows don't mix well

### ISSUE THREAD

**Issue by hertg** [leftwm/leftwm#517]
**Describe the bug**
I recently changed `focus_behaviour` to `ClickTo`, now I get weird freezes occasionally, which can only be resolved by switching tags via keybinding.

**To Reproduce**
I can't provide a 100% working step-by-step to reproduce it, because I still haven't found out which of my actions actually cause the bug.

Here is how I was able to reproduce it on at least two different computers within a few minutes of use:
1. Change `focus_behaviour` to `ClickTo`
2. Toggle one or more windows to be floating
3. Move floating windows around and resize them
4. The issue will eventually happen, sometimes right after toggling a window to be floating, sometimes after 20-30 seconds of moving stuff around

Something I observed when the issue occurs:
I use [unclutter](https://wiki.archlinux.org/title/Unclutter), which hides my mouse cursor after I haven't moved it around for some time. Funny enough, as soon as the issue occurs, I can still move the mouse cursor around and see it moving, but it will suddenly disappear because unclutter thinks it isn't moving.. Maybe that helps in tracking down the issue.

**Expected behavior**
No freezes :)

**Screenshots**
*None*

**Environment:**
 - OS/Distro: Arch
 - Output of `leftwm-check` (I'm on latest master)
   ```
   :: LeftWM version: 0.2.9
   :: LeftWM git hash: b4a0686
   :: Loading configuration . . .
       -> Configuration loaded OK
   :: Checking keybinds . . .
       -> All keybinds OK
   :: Checking environment . . .
       -> Environment OK
   :: Checking theme . . .
       -> Theme OK
   ```

**Additional context**
*None*

**Comment by hertg** [comment]
I've also experienced this issue in `Sloppy` in the last few days/weeks. When moving/resizing floating windows (mostly popups) leftwm freezes occasionally, most of the time it catches up and applies the move/resize requests after a few seconds and continues normally, very rarely I need to SoftReload to unfreeze it though.

**Comment by VuiMuich** [comment]
So I guess #585 might be related, right?

**Comment by hertg** [comment]
> So I guess #585 might be related, right?

I don't think so. This one here happens with all kinds of windows. The comment about it being "mostly popups" is probably misleading because popups and my alacritty scratchpad are basically the only floating windows I use.

**Comment by AethanFoot** [comment]
I have found one repeatable method demonstrated in the video:

https://user-images.githubusercontent.com/63980259/143466656-ec378a42-156f-4f50-bbd8-fff63be6d07c.mp4

I drag the second window into floating, retile it, click the new second window, then use the MoveWindowTop command. This then freezes until I click a new window.

**Comment by AethanFoot** [comment]
Ignore those it seems to be an issue #587

**Comment by AethanFoot** [comment]
One way to at least consistently trip it up is to hold your mousekey and click the moving and/or resizing key over and over. It seems to break very quickly.

**Comment by VuiMuich** [comment]
This seems to be mostly fixed by #587 but as Aethan stated, it might still be possible to trigger this.

So we keep this open and move it to the `0.2.11` milestone

**Comment by VuiMuich** [comment]
Lets assume #696 solved this. If not, just reopen :man_shrugging:
